101-hdinsight-linux-ssh-publickey-metastore-vnet/azuredeploy.json

```diff
@@ -158,8 +157,9 @@
               "hive_database_name": "[parameters('existingHiveMetastoreDatabaseName')]",
               "hive_database_type": "mssql",
               "hive_existing_mssql_server_database": "[parameters('existingHiveMetastoreDatabaseName')]",
-              "hive_existing_mssql_server_host": "[parameters('existingHiveMetastoreServerName')]",
-              "hive_hostname": "[parameters('existingHiveMetastoreServerName')]"
+              "hive_existing_mssql_server_host": "[reference(resourceId('Microsoft.Sql/servers',parameters('existingHiveMetastoreServerName')), '2018-06-01-preview').fullyQualifiedDomainName]",
```

Comment:
need to include the resourceGroupName parameter in the resourceId function - otherwise this will fail if the SQL Server is in a separate resourceGroup